README.md

```diff
@@ -9,7 +9,7 @@ a suite of tests against your systems gives you the confidence to _greenlight_
 them for promotion to production. The primary goals of this framework are:
 
 - Steps should be _composable_ to drive down repetition in similar tests.
-- TODO: Tests should be _distributable_ and support parallelization out-of-the-box.
+- Tests should be _distributable_ and support parallelization out-of-the-box.
```

Comment:
Hmm - parallelization is accurate, but technically we still haven't made tests distributable. Might be worth removing that word and adding a separate issue for distribution across multiple agents.